Workbooks/Azure Active Directory/Auth Methods Activity/authmethodsactivity.workbook

```diff
@@ -0,0 +1,528 @@
+                  "timeContext": {
+                    "durationMs": 86400000
+                  }
+                },
+                {
+                  "id": "2c5d1b67-b8c9-47d3-a9bc-fcd389bcad31",
+                  "version": "KqlParameterItem/1.0",
+                  "name": "Methods",
+                  "type": 2,
+                  "isRequired": true,
+                  "multiSelect": true,
+                  "quote": "'",
+                  "delimiter": ",",
+                  "value": [
+                    "value::all"
+                  ],
+                  "typeSettings": {
+                    "additionalResourceOptions": [
+                      "value::all"
+                    ],
+                    "showDefault": false
+                  },
+                  "jsonData": "[\r\n    {\"value\": \"Password\", \"label\": \"Password\"},\r\n    {\"value\": \"Text message\", \"label\": \"Text message\"},\r\n    {\"value\": \"Mobile app notification\", \"label\": \"Mobile app notification\"},\r\n    {\"value\": \"Mobile app verification code\", \"label\": \"Mobile app verification code\"},\r\n    {\"value\": \"Phone call (Authentication phone)\", \"label\": \"Phone call\"},\r\n    {\"value\": \"Passwordless phone sign-in\", \"label\": \"Passwordless phone sign-in\"},\r\n    {\"value\": \"WindowsHelloForBusiness\", \"label\": \"Windows Hello for Business\"},\r\n    {\"value\": \"TemporaryAccessPass\", \"label\": \"Temporary Access Pass\"},\r\n    {\"value\": \"Sms\", \"label\": \"SMS Sign-in\"},\r\n    {\"value\": \"OATH hardware token\", \"label\": \"OATH hardware token\"}\r\n]",
+                  "timeContext": {
+                    "durationMs": 86400000
+                  }
+                }
+              ],
+              "style": "pills",
+              "queryType": 0,
+              "resourceType": "microsoft.operationalinsights/workspaces"
+            },
+            "name": "parameters - 2"
+          },
+          {
+            "type": 3,
+            "content": {
+              "version": "KqlItem/1.0",
+              "query": "SigninLogs\r\n| where TimeGenerated > ago(30d)\r\n| project AuthenticationDetails\r\n| where AuthenticationDetails contains \"authenticationMethod\"\r\n| extend AuthenticationDetails = parse_json(AuthenticationDetails)\r\n| mv-expand AuthenticationDetails\r\n| project method = case(tostring(AuthenticationDetails.authenticationMethod) == \"PHS\", \"Password\", tostring(AuthenticationDetails.authenticationMethod) == \"PHS, StagedRollout\", \"Password\", tostring(AuthenticationDetails.authenticationMethod) == \"CloudOnlyPassword\", \"Password\", tostring(AuthenticationDetails.authenticationMethod))\r\n| where method != \"\"\r\n| where \"{Methods}\" has method\r\n| summarize count() by method\r\n",
```

Comment:
Once AzureMonitor has the new changes for authentication method "friendly names", this will need to be updated. (Example, we no longer will have CloudOnlyPassword but just Password).